cluster/stone-prd-rh01/tenants/rh-trex-tenant/integration_test_scenario.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: remotesecret-enterprise-contract
+  namespace: rh-trex-tenant
+spec:
+  application: remotesecret
```

Comment:
I think you are including an ITS for another tenant's application?

Reply:
Removing!